iset.mm

```diff
@@ -111475,6 +111475,23 @@ seq m ( + , ( n e. ZZ |-> if ( n e. A , [_ n / k ]_ B , 0 ) ) , CC )
     $}
   $}
 
+  ${
+    $d F k v $.  $d M k v $.  $d Z k $.  $d k ph v $.
+    zsum.1 $e |- Z = ( ZZ>= ` M ) $.
+    zsum.2 $e |- ( ph -> M e. ZZ ) $.
+    isum.3 $e |- ( ( ph /\ k e. Z ) -> ( F ` k ) = B ) $.
+    isum.4 $e |- ( ( ph /\ k e. Z ) -> B e. CC ) $.
+    $( Series sum with an upper integer index set (i.e. an infinite series).
```

Comment:
Shouldn't it be something like "The sum of a series is equal to its limit. " ?

Reply:
It's the limit of the partial sums, but usually I don't try to wordsmith stuff like this when I'm just copying it from set.mm to iset.mm.